app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command = command.split(" ")
+    file_name = command[1]
```

Comment:
Consider using more descriptive variable names, such as `source_file_name` and `destination_file_name`, to improve code readability and adhere to the checklist's code style guidelines.